package.json

```diff
@@ -21,6 +21,7 @@
   "homepage": "https://github.com/metamaps/metamaps#readme",
   "dependencies": {
     "ajaxq": "0.0.7",
+    "async": "2.5.0",
```

Comment:
if this is only used for `apply`, I believe there's a built in we can use instead

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply

or I think actually the way you're currently using apply is more like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call

Reply:
wow, this is actually different than I thought. It would be nice if you can somehow make this work though to avoid the dependency.

Reply:
The way that this version of `apply` works is different than either of those
https://caolan.github.io/async/docs.html#apply
I used it a lot at bitHound that's how I knew about it. Really good for functional js

Reply:
@devvmh what's the advantage in your opinion of avoiding the dependency?

Reply:
If its useful and good lets add it.

Someday ill set up a profiler and look at our bundle size, then we can make decisions about including dependencies with actual data